backend/trojan_dining/views.py

```diff
@@ -81,7 +83,34 @@ def post(self, request):
                 return HttpResponse(400)
             
             try:  
-                Subscription(item_id = item_id, email = email, phone_no = phone_number).save()
+                try:
+                    Subscription.objects.get(item_id = item_id, email = email, phone_no = phone_number)
+                except:
+                    Subscription(item_id = item_id, email = email, phone_no = phone_number).save()
                 return HttpResponse(201)
             except:
                 return HttpResponse(500)
+
+class GetSubscriptions(APIView):
+    def get(self, request):
+        user_email = request.GET.get('email')
+
+        if user_email is None:
+            return HttpResponse(status = 400)
+        else:
+            subscription_list = list(Subscription.objects.filter(email = user_email).values())
+
+            for sub in subscription_list:
+                sub['name'] = MenuItem.objects.get(item_id = sub['item_id']).name
+
+            return JsonResponse({"Subscriptions": subscription_list})
+
+class DeleteSubscriptions(APIView):
+    def delete(self, request):
+        body = json.loads(request.body)
+        subscriptions = body['subscriptions']
```

Comment:
Can you define the API schema for this somewhere? Perhaps in our API spec/discord/a comment

Reply:
I'll update the google docs

Reply:
:duckpray: